backend/authServer.js

```diff
@@ -0,0 +1,108 @@
+require("dotenv").config();
```

Comment:
wasn't this file already added in the previous PR? or is it because this branch is based off of main and doesn't include changes in the previous PR: routing sign login page?

Reply:
Yes that is correct. This is why i was asking questions as to how to branch off while waiting for requests to be approved. 

The reason why this PR has so many file changes is like you said, its because i made a branch of main and main still has none of the sign up auth code.